tests/createWcsWithSip.py

```diff
@@ -79,6 +78,7 @@ def testBigXy0(self):
             src.set(yKey, src.get(yKey) - 500 + y0)
         bbox = afwGeom.Box2I(afwGeom.Point2I(x0, y0), afwGeom.Extent2I(1000, 1000))
         res = self.astrom.determineWcs2(cat, bbox=bbox)
+        self.assertTrue(res.sipWcs is not None, "Failed to fit SIP terms")
```

Comment:
Is the error message sufficiently clearer to justify that change? I am keen on assertLess(a, b) instead of assertTrue(a<b) and the like because they give better error messages and are easier to read. I'm not sure I see the value in this one.